docs/PIPELINE_DEPLOYMENTS.md

```diff
@@ -0,0 +1,169 @@
+- Confidential project: `cmek_reidentification_crypto_key`
+
+### Disable Public IPs
+
+[Disabling Public IPs helps to better secure you data processing infrastructure.](https://cloud.google.com/dataflow/docs/guides/routes-firewall#turn_off_external_ip_address).
+Make sure you have your subnetwork configured as [Subnetwork section](#subnetwork) details.
+
+### Enable Streaming Engine
+
+Enabling Streaming Engine it is important to ensure all the performance benefits of the infrastructure. You can learn more about it in the [documentation](https://cloud.google.com/dataflow/docs/guides/deploying-a-pipeline#streaming-engine).
+
+## Deploying Dataflow Flex Jobs
+
+We recommend the usage of [Flex Job Templates](https://cloud.google.com/dataflow/docs/guides/templates/using-flex-templates).
+You can learn more about the differences between Classic and Flex Templates [here](https://cloud.google.com/dataflow/docs/concepts/dataflow-templates#evaluating-which-template-type-to-use).
+
+### Deploying with Terraform
+
+Use the Dataflow Flex Job Template [submodule](../modules/dataflow-flex-job/README.md).
+
+### Deploying with `gcloud` Command
+
+You can run the following commands to create a **Java** Dataflow Flex Job using the **gcloud command**:
+
+```sh
+
+export PROJECT_ID=<PROJECT_ID>
+export DATAFLOW_BUCKET=<DATAFLOW_BUCKET>
+export DATAFLOW_KMS_KEY=<DATAFLOW_KMS_KEY>
+export SERVICE_ACCOUNT_EMAIL=<SERVICE_ACCOUNT_EMAIL>
+export SUBNETWORK=<SUBNETWORK_SELF_LINK>
+
+gcloud dataflow flex-template run "TEMPLATE_NAME`date +%Y%m%d-%H%M%S`" \
+    --template-file-gcs-location="TEMPLATE_NAME_LOCATION" \
+    --project="${PROJECT_ID}" \
+    --staging-location="${DATAFLOW_BUCKET}/staging/" \
+    --temp-location="${DATAFLOW_BUCKET}/tmp/" \
+    --dataflow-kms-key="${DATAFLOW_KMS_KEY}" \
+    --service-account-email="${SERVICE_ACCOUNT_EMAIL}" \
+    --subnetwork="${SUBNETWORK}" \
+    --region="us-east4" \
+    --disable-public-ips \
+    --enable-streaming-engine
+
+```
+
+For more details about `gcloud dataflow flex-template` see the command [documentation](https://cloud.google.com/sdk/gcloud/reference/dataflow/flex-template/run).
+
+## Common tasks
+
+### How do I rerun the Dataflow flex pipeline for new data?
+
+#### Streaming Dataflow
+
+##### Pub/Sub to Bigquery
+
+If you are using our [Python De-Identification Flex Template sample](../flex-templates/java/regional_dlp_de_identification/README.md), you can provide a topic as value for the `input_topic` parameter.
+
+Examples using Python De-Identification Flex Template:
+
+- [Regional DLP](../examples/regional-dlp/README.md)
+
+If you are using our [Regional DLP Example](../examples/regional-dlp/README.md), the Dataflow deployed is
+already waiting for new input data in the topic. You just need do publish a new message with the format
+expected by the template. You can check how publishing messages [here](https://cloud.google.com/pubsub/docs/publisher).
```

Comment:
might also provide a tip on what is the topic name to use that we create for them. I think a link to the code example is fine.